src/lib/Invitation/Persistence/Handler.php

```diff
@@ -0,0 +1,85 @@
+<?php
+
+/**
+ * @copyright Copyright (C) Ibexa AS. All rights reserved.
+ * @license For full copyright and license information view LICENSE file distributed with this source code.
+ */
+declare(strict_types=1);
+
+namespace Ibexa\User\Invitation\Persistence;
+
+use Ibexa\Contracts\User\Invitation\Invitation;
+use Ibexa\Core\Base\Exceptions\NotFoundException;
+
+class Handler
```

Comment:
Could you please extract interface ? API should not directly refer to Persistence implementation.